.buildkite/filebeat/filebeat-pipeline.yml

```diff
@@ -178,7 +178,37 @@ steps:
               debug: true
         notify:
           - github_commit_status:
-              context: "filebeat: Go Integration Tests"
+              context: "filebeat: Go Integration Tests / Ubuntu x86_64"
+
+      - label: ":windows: Filebeat: Go Integration Tests"
```

Comment:
What's the reason we don't use a [buildkite matrix](https://buildkite.com/docs/pipelines/configure/workflows/build-matrix)?

Reply:
Can you double-check? I found it complicated to use the same snippets without taking advantage of some features, such as matrix.